package.json

```diff
@@ -22,16 +22,17 @@
     "test": "grunt test"
   },
   "dependencies": {
+    "amqplib": "^0.5.1",
     "async": "1.5.2",
+    "body-parser": "1.15.0",
     "dateformat": "1.0.12",
     "express": "^4.11.2",
-    "body-parser": "1.15.0",
-    "amqplib": "^0.5.1",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "https://github.com/dcalvoalonso/iotagent-node-lib.git#task/exportConfigNgsi2",
```

Comment:
I think this should use `#task/activeAttsNgsi2`, isn't it?

Reply:
Fixed

Reply:
It would be great if you could add the git hash along with the "fixed" mark, e.g:

Fixed in 6147e04

That way each comment is traceable to the commit where it was addressed which is very useful, specially if the PR is long as has many comments.

Thanks!